uint8 id_c
uint8 id_t
uint8 id_n
uint8 id_f
uint8 len
uint8[8] data
time time
